dash/src/util/LocalityDomain.cc

```diff
@@ -35,12 +35,6 @@ std::ostream & operator<<(
 // Public Constructors
 // -------------------------------------------------------------------------
 
-
-dash::util::LocalityDomain::LocalityDomain()
-: _domain(nullptr),
-  _subdomains(nullptr)
-{ }
-
 dash::util::LocalityDomain::LocalityDomain(
```

Comment:
I could not reproduce this, but perhaps I fixed it along the way. Does it work for you now?